fili-core/src/test/groovy/com/yahoo/bard/webservice/metadata/DataSourceMetadataServiceSpec.groovy

```diff
@@ -19,23 +15,39 @@ class DataSourceMetadataServiceSpec extends BaseDataSourceMetadataSpec {
         JerseyTestBinder jtb = new JerseyTestBinder()
         PhysicalTableDictionary tableDict = jtb.configurationLoader.getPhysicalTableDictionary()
         DataSourceMetadataService metadataService = new DataSourceMetadataService()
-
         DataSourceMetadata metadata = new DataSourceMetadata(tableName, [:], segments)
 
         when:
         metadataService.update(tableDict.get(tableName), metadata)
 
         then:
-        metadataService.allSegments.get(tableDict.get(tableName)) instanceof AtomicReference
+        metadataService.allSegmentsByTime.get(tableDict.get(tableName).getTableName()) instanceof AtomicReference
+        metadataService.allSegmentsByColumn.get(tableDict.get(tableName).getTableName()) instanceof AtomicReference
```

Comment:
readability might be improved by pulling tableDict.get(tableName) into a local variable.